src/main/docker/docker-compose.yml

```diff
@@ -1,21 +1,11 @@
-version: '2.4'
 services:
     artemis-app:
         platform: linux/x86_64
         image: ghcr.io/ls1intum/artemis
         build:
             context: ../../..
-            dockerfile: src/main/docker/Dockerfile
-        volumes:
-            - ../resources/config/application-dev.yml:/opt/artemis/application-dev.yml:ro
-            - ../resources/config/application-artemis.yml:/opt/artemis/application-artemis.yml:ro
-        environment:
-            _JAVA_OPTIONS: -Xmx5120m -Xms2560m
-            SPRING_PROFILES_ACTIVE: dev,bamboo,bitbucket,jira,artemis,scheduling,athene,local
-            SPRING_DATASOURCE_URL: jdbc:mysql://artemis-mysql:3306/Artemis?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
-            SPRING_DATASOURCE_USERNAME: root
-            SPRING_DATASOURCE_PASSWORD:
-            JHIPSTER_SLEEP: 30 # gives time for other services to boot before the application
+            dockerfile: src/main/docker/artemis/Dockerfile
+        env_file: .env
```

Comment:
Just for information. The current .env is for the development version and not for production. Should we maybe rename it to dev.env? Also I'm not sure how we like to manage the inheritance of env files. Copying them is not ideal in my opinion.

Reply:
Added multiple env files to manage inheritance.